README.md

```diff
@@ -1,162 +1,263 @@
+-   **Multiple-outputs workflow**: When the number of portfolios and/or
+    investors is not exactly one, we use a different set of optimized
+    scripts. This workflow is typically used to produce static (.pdf)
+    reports. By contrast with the so-called "online" workflow, this one
+    has been usually referred to as "the offline workflow"; again, this
+    is misleading.
+
+Both workflows include three steps:
+
+1.  Clean portfolio: Clean the input portfolio and merge in financial
+    data, to categorize each holding and identify whether it's equity
+    (EQ) or corporate bonds (CB).
+2.  Run Analysis: Merge portfolios with asset-level data and scenarios,
+    then group results at company, portfolio and regional level.
+3.  Present results: Present the results in a clear output format.
+
+--
+
+Note that "`$`" and "`#>`" precede bash and R output, respectively.
+
+Siblings
+--------
+
+The PACTA family includes these siblings: PACTA\_analysis,
+StressTestingModelDev, create\_interactive\_report, pacta-data. They are
+all available at <https://github.com/2DegreesInvesting> and some
+workflows assume they all live directly under the same parent directory.
+Before running a workflow, you may pull the latest commits of the
+relevant siblings from Github, to ensure the code you use is up to date.
+
+Computing environment
+---------------------
+
+A functional computing environment is available in [this Docker
+image](https://github.com/2DegreesInvesting/docker/tree/master/r-packages)
+(how to [install Docker](https://docs.docker.com/engine/install/)). For
+an example using that image let's render the document you are now
+reading, using all PACTA siblings, and personal configuration files from
+the home directory in the host computer (my laptop). The entire
+computing environment is captured in the file "docker-compose.yml":
+
+    version: "0.0.0.9000"
+    services:
+      pacta:
+        image: 2dii/r-packages
+        working_dir: /root/PACTA_analysis
+        volumes:
+          # The ssh protocol is required when 2FA is enabled (recommended)
+        - ~/.ssh:/root/.ssh
+          # Access GITHUB_PAT and other personal secrets
+        - ~/.Renviron:/root/.Renviron
+          # Avoid re-configuring git
+        - ~/.gitconfig:/root/.gitconfig
+          # Read/write PACTA siblings in the host from the container
+        - ../PACTA_analysis:/root/PACTA_analysis
+        - ../pacta-data:/root/pacta-data
+        - ../create_interactive_report:/root/create_interactive_report
+        - ../StressTestingModelDev:/root/StressTestingModelDev
+
+We can now `run` an ephemeral (`--rm`) container of the `pacta` service
+with `docker-compose` (how to [install
+docker-compose](https://docs.docker.com/compose/install/)):
+
+    docker-compose run --rm pacta
+
+From the container's `working_dir`ectory we now start an `R` session and
+`render()` the file you are now reading:
+
+    R
+    rmarkdown::render("README.Rmd", output_format = "md_document")
+
```

Comment:
@AlexAxthelm, I have little experience with Docker and less with `docker-compose`. Do you have any comment about this bit?